siteroot/settings/base.py

```diff
@@ -46,6 +46,7 @@
 ]
 
 MIDDLEWARE = [
+    'bookmarks.middlewares.HealthCheckMiddleware',
```

Comment:
A Django view would be a better option, but let's resolve the previous comment first.

Reply:
To convert this into a view:
- add a new `health.py` file to `bookmarks/views`
- add a function there, implementation should be everything within the `if request.path == '/health':` block, the condition itself is not needed
- register the view function in `bookmarks/urls.py`

Also would be great to have at tests for the view (healthy / unhealthy cases).

Reply:
Thanks @sissbruecker this is super helpful! I went ahead and converted it to a view. 

I also added a simple test for the healthy case. I wasn't sure how I could simulate the DB being down in a test. Any ideas?

Reply:
Added a test for the unhealthy scenario.